docs/using-radicle/tracking-and-viewing.md

```diff
@@ -4,10 +4,10 @@ title: Tracking and viewing contributions
 ---
 
 On Radicle, peers track projects they are interested in. If a peer wants to
-contribute to a project, they fork it and create their own view of the project
-that they publish changes to. 
+contribute to a project, they fork it, create their own view of the project
+that they publish changes to and then package those changes as a patch.
 
-Therefore, to view someone else's changes to your project, you first need to add
+Therefore, to view someone else's patches to your project, you first need to add
```

Comment:
Proposal:

Therefore, to view someone else's patch to your project, you first need to add […]